src/view/com/composer/Composer.tsx

```diff
@@ -1213,17 +1213,30 @@ export function useComposerCancelRef() {
   return useRef<CancelRef>(null)
 }
 
-function useAnimatedBorders() {
+function useScrollTracker({
+  scrollViewRef,
+  stickyBottom,
+}: {
+  scrollViewRef: AnimatedRef<Animated.ScrollView>
+  stickyBottom: boolean
+}) {
   const t = useTheme()
-  const hasScrolledTop = useSharedValue(0)
-  const hasScrolledBottom = useSharedValue(0)
   const contentOffset = useSharedValue(0)
   const scrollViewHeight = useSharedValue(Infinity)
   const contentHeight = useSharedValue(0)
 
-  /**
-   * Make sure to run this on the UI thread!
```

Comment:
This was already called from JS thread in some cases, I adjusted https://github.com/bluesky-social/social-app/pull/6045/commits/51c129fab5ab93e9ed0bcf9dfea8a30f0743b149 to reflect. It's also fine actually aside from how values are read, https://github.com/bluesky-social/social-app/pull/6045/commits/a1b6af01e4009914b828c3617b2633da32deca6c took care of the only bit that was off.